### DISCUSSION TITLE
Does Ozone currently support heterogeneous storage？

### DISCUSSION THREAD

**Discussion by Misszhangruihua** [apache/ozone#5665] [General]
We use Ozone as the underlying architecture for object storage, and we want to achieve heterogeneous storage functions similar to HDFS. We store Ozone data at different times based on time or other hot and cold architectures, and store different data on different disks?

**Comment by arp7** [comment]
Hi @Misszhangruihua , Ozone currently doesn't have heterogeneous storage support. This is a roadmap item for the project, any community contributions to help build this feature will be welcome!
